src/docs/blueprint_documentation.md

```diff
@@ -143,6 +143,10 @@ isBound(AboutDialog) ? rebind(AboutDialog).to(TheiaBlueprintAboutDialog).inSingl
 
 To use another custom dialog widget, remove this code, extend Theia’s AboutDialog class, and (re)bind it as above.
 
+### Customizing the Preferences
+
+The default preferences directory in Eclipse Theia Blueprint is `.theia-blueprint` and located as described in the [Preferences documentation](https://theia-ide.org/docs/preferences/). You can customize this location by modifying `theia-extensions/theia-blueprint-product/src/node/theia-blueprint-variables-server.ts`
```

Comment:
I had initially written the path out of keeping things the same with lines earlier in the doc that similarly write the path out but don't link, such as: https://github.com/eclipse-theia/theia-website/blob/8fcea7d9bfa6b6acdda5b358dc4e57c0b752f84b/src/docs/blueprint_documentation.md?plain=1#L138

Perhaps using links instead of paths should be a separate PR that covers other places where needed?

Reply:
I see, but I don't think there is a strong reason to not add the link for this new documentation now, and open a follow-up issue for the other links. It is not immediately clear for readers where this location exists and a link would provide more clarity and usability.

Reply:
ok, went ahead and made it a link.